### ISSUE TITLE
Windows 10 C# SPXERR_AUDIO_SYS_LIBRARY_NOT_FOUND

### ISSUE THREAD

**Issue by Brandon689** [Azure-Samples/cognitive-services-speech-sdk#1851]
**Describe the bug**

**Version of the Cognitive Services Speech SDK**
1.25.0

**Platform, Operating System, and Programming Language**
 - Windows 10
 -  x64
 - C#

**Additional context**
If I make any project with test code for Text to Speech it gives this error. If I do nothing except move the non functioning project into the azure samples solution project folder and run it inside it, it works normally.

**Comment by pankopon** [comment]
@Brandon689 Can you test if the following works on your machine? This was verified on Windows 10 22H2 (OS Build 19045.2604), with Visual Studio 2022 Version 17.5.0 and the latest Visual C++ redist for Visual Studio 2015-2022 (from https://learn.microsoft.com/cpp/windows/latest-supported-vc-redist?view=msvc-170) installed.

- create new C# Console App (multi-platform, not .NET Framework / Windows specific)
- use the default Location (somewhere under your user folder probably)
- choose Framework .NET 6.0 (LTS)
- on VS, go to *Tools - NuGet Package Manager - Manage NuGet Packages for Solution*, then
*Browse*, find and select *Microsoft.CognitiveServices.Speech*, Version 1.25.0; install to the project
- replace the default code with the following (use your subscription key and service region), build and run
```
using System;
using System.Threading.Tasks;
using Microsoft.CognitiveServices.Speech;

namespace helloworld
{
    class Program
    {
        public static async Task SynthesisToSpeakerAsync()
        {
            var config = SpeechConfig.FromSubscription("YourSubscriptionKey", "YourServiceRegion");
            string text = "This is an example of speech synthesis.";

            using var synthesizer = new SpeechSynthesizer(config);
            using var result = await synthesizer.SpeakTextAsync(text);

            if (result.Reason == ResultReason.Canceled)
            {
                var cancellation = SpeechSynthesisCancellationDetails.FromResult(result);
                Console.WriteLine($"CANCELED: Reason={cancellation.Reason}");

                if (cancellation.Reason == CancellationReason.Error)
                {
                    Console.WriteLine($"CANCELED: ErrorCode={cancellation.ErrorCode}");
                    Console.WriteLine($"CANCELED: ErrorDetails=[{cancellation.ErrorDetails}]");
                }
            }
        }

        static async Task Main()
        {
            await SynthesisToSpeakerAsync();
            Console.WriteLine("Press any key to exit...");
            Console.ReadKey();
        }
    }
}
```

**Comment by Brandon689** [comment]
😗 After you said "use the default Location" I had a realization. This was caused by my containing folders not being in English. Its in Japanese. So moving it into the samples folder was just fixing it due to that.

I had some different problem months ago that was caused by this folder naming, I forget what it was. But All my previous Cognitive Services apps are in this same non-English folder, that I developed awhile ago, so it didn't used to error.
